graylog2-web-interface/docs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@graylog/frontend-documentation",
+  "version": "1.0.0",
+  "description": "Graylog Frontend Documentation",
+  "main": "index.js",
```

Comment:
It seems we do not have an `index.js` file, but since this is the default when no `main` entry is defined and there is no good alternative, I do not have a better suggestion.